packages/api-specification/src/window-api.html

```diff
@@ -38,13 +38,30 @@
     <button type="button" id="new-window-test" class="btn btn-default">Open Window</button>
   </form>
 
+  <hr/>
+
+  <h4>New window controls</h4>
+  <form class="form-inline">
+    <button id="close-window" type="button" class="btn btn-default">Close</button>
+    <button id="show-window" type="button" class="btn btn-default">Show</button>
+    <button id="hide-window" type="button" class="btn btn-default">Hide</button>
+    <button id="focus-window" type="button" class="btn btn-default">Focus</button>
+    <button id="blur-window" type="button" class="btn btn-default">Blur</button>
+  </form>
+
   <h3>Overview</h3>
 
   <p>The Window API is used to create a new browser window.</p>
 
   <h3>Constructor</h3>
 
   <pre>
+  new ssf.Window()
+  </pre>
+
+  <p>Creates a new window object for the current window.</p>
```

Comment:
Ah, I can see why that would be useful!

From an API perspective, when you call `new ssf.Window()` you really expect to get a 'new' thing. I think this would make more sense as a 'static' method, how about this - `ssf.Window.getCurrent()`